sunpy/util/xml.py

```diff
@@ -124,3 +123,66 @@ def get_node_text(node):
         else:
             raise NotTextNodeError
     return t
+
+def xml_comments_to_dict(xmlstring):
+    """
+    Parses all the XML comments provided in an XML string into a dictionary
+
+    .. warning::
+        This method does not support multiple values of comments for elements
+        of the same name but with different values. It always takes the last value.
+
+    Parameters
+    ----------
+    xmlstring : `str`
+        A `str` of xml content.
+
+    Returns
+    -------
+    `dict`:
+        The comments present in the string xml input as a dictionary.
+    Examples
+    --------
+    ::
+
+        <outer>
+            <inner comment = "One">one</inner>
+            <inner comment = "Two">two</inner>
+        </outer>
+
+    gives you the dict::
+
+        {u'inner': 'Two'}
+    """
+    key_comments_dict = {}
+    history = []
+    node_comments_to_dict(parseString(xmlstring), key_comments_dict, history)
+    history = "".join(history).strip()
+    return key_comments_dict, history
+
+
+def node_comments_to_dict(node, dic, history):
```

Comment:
Can dic not be renamed to a more useful name?

Reply:
changed it to comments`